src/main/asciidoc/_chapters/architecture.adoc

```diff
@@ -260,6 +260,73 @@ For region name, we only accept `byte[]` as the parameter type and it may be a f
 
 Information on non-Java clients and custom protocols is covered in <<external_apis>>
 
+[[client.masterregistry]]
+=== Master registry (new as of release 3.0.0)
```

Comment:
New as of 2.3.0, right? I think you'll want to update this throughout the doc.

Reply:
I wanted to update the docs once it actually makes it to the 2.3.0...that ok? (didn't want to do that without committing to the branch).